test/Hpack/ConfigSpec.hs

```diff
@@ -815,8 +815,9 @@ spec = do
           touch "src/Foo.hs"
           touch "src/Bar.hs"
           touch "src/Baz.lhs"
+          touch "src/Qux.hsc"
           )
-          (map (executableOtherModules . sectionData) . packageExecutables >>> (`shouldBe` [["Bar", "Baz", "Foo"]]))
+          (map (executableOtherModules . sectionData) . packageExecutables >>> (`shouldBe` [["Bar", "Baz", "Foo", "Qux"]]))
```

Comment:
At the point we have unit tests for this, we don't need to test all three cases up here anymore.  just doing it for e.g. `.hs` is fine.